editor.planx.uk/src/@planx/components/MultipleFileUpload/Editor.tsx

```diff
@@ -71,3 +119,159 @@ function MultipleFileUpload(props: Props) {
     </form>
   );
 }
+
+function FileTypeEditor(props: ListManagerEditorProps<FileType>) {
+  const isConditionalRule = checkIfConditionalRule(props.value.rule.condition);
+
+  return (
+    <Box sx={{ flex: 1 }} data-testid="rule-list-manager">
+      <ModalSubtitle title="File" />
+      <InputRow>
+        <Input
+          required
+          name="key"
+          value={props.value.key}
+          onChange={(e) =>
+            props.onChange(merge(props.value, { key: e.target.value }))
+          }
+          placeholder="File type"
+        />
+      </InputRow>
+      <InputRow>
+        <Input
+          required
+          name="fn"
+          value={props.value.fn}
+          onChange={(e) =>
+            props.onChange(merge(props.value, { fn: e.target.value }))
+          }
+          placeholder="Data field"
+        />
+      </InputRow>
+      <ModalSubtitle title="Rule" />
+      <InputRow>
+        <SelectInput
+          value={props.value.rule.condition}
+          onChange={(e) =>
+            props.onChange(
+              setCondition(
+                Condition[e.target.value as keyof typeof Condition],
+                props.value
+              )
+            )
+          }
+        >
+          {Object.entries(Condition).map(([key, value]) => (
+            <MenuItem key={key} value={value}>
+              {upperFirst(lowerCase(value))}
```

Comment:
nit: does lodash `capitalize` accomplish the same here but in one method instead of two? https://lodash.com/docs/4.17.15#capitalize